src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+  const longtermDiscount = 50;
```

Comment:
you have typo in those two variables names, please use camel case, `longTermDiscount`